nucleus/pom.xml

```diff
@@ -138,8 +138,8 @@
         <javax.el.version>3.0.1-b08</javax.el.version>
         <javax.el-api.version>3.0.1-b04</javax.el-api.version>
         <saaj-api.version>1.4.0</saaj-api.version>
-        <hk2.version>2.5.0-b44</hk2.version>
-        <hk2.plugin.version>2.5.0-b44</hk2.plugin.version>
+        <hk2.version>2.5.0-b57</hk2.version>
```

Comment:
The latest release of HK2 is 2.5.0-b60, you should be using that.

Reply:
I am using the version after removing HK2 modules from HK2 code. May be we can create an issue and update the hk2 versions. What do you think?